shared/cluster.yaml

```diff
@@ -0,0 +1,4 @@
+cluster_module_load: slurm
+snakemake_cluster_cmd: SlurmEasy --mem-per-cpu {cluster.memory} --threads {threads} --log
+__default__:
+  memory: 6G
```

Comment:
Wouldn't it make more sense to set this per rule? Or is the `__default__` doing some sort of magic such that putting this somewhere under a rule overrides it? My presumption would otherwise be that this should be `{params.memory}`, which would then need to be set throughout (presumably with some presets defined in a similar yaml file).

Reply:
Looking further, it appears that the default is indeed overridden as appropriate by tools. If that's correct then excellent!

Reply:
yeah the __default__ acts as global default for memory. However, it seems it can be only overwritten by an entry in the provided --cluster-config yaml (cluster.yaml in current RNA-seq) and not within the rule by eg cluster: memory=10G or params: .,.